platypus/core.py

```diff
@@ -1234,6 +1234,21 @@ def normalize(solutions, minimum=None, maximum=None):
 
     return minimum, maximum
 
+def objectives_normalized_values(solutions, minimum=None, maximum=None):
+    
+    """Normalizes the solution objectives.
+
+    Normalizes the objectives of each solution within the minimum and maximum
+    bounds.  If the list of minimum and maximum bounds for objectives are not provided, then the
+    bounds are computed based on the bounds of the solutions.
+    """
+
+    normalize(solutions, minimum=None, maximum=None)
+
+    normalized_values = [s.normalized_objectives for s in solutions]
```

Comment:
Also note that `normalize(...)` does not apply to infeasible solutions.  So if the list of solutions contains any that violate constraints, I think this would throw a `'Solution' object has no attribute 'normalized_objectives'`.  And it's not guaranteed the result of an algorithm filters out such infeasible solutions, as it's up to the implementation.

Perhaps a more generalized solution is to have several methods that can be chained together:

```
objs = normalized_objectives(feasible(unique(algorithm.result)))
```